CMakeLists.txt

```diff
@@ -189,3 +190,4 @@ if (WIN32)
 endif (WIN32)
 
 install(TARGETS qmc-example RUNTIME DESTINATION ${CMAKE_INSTALL_BINDIR})
+install(FILES ${CMAKE_CURRENT_BINARY_DIR}/QMatrixClient.pc DESTINATION ${CMAKE_INSTALL_LIBDIR}/pkgconfig)
```

Comment:
It is only worth installing on Linux, isn't it?

Reply:
True. Although windows supports pkg-config, no one uses that.